src/app/storage.js

```diff
@@ -49,7 +49,7 @@ function Storage () {
   safeKeys().forEach(function (name) {
     if (name.indexOf('sol-cache-file-', 0) === 0) {
       var content = window.localStorage.getItem(name)
-      window.localStorage.setItem(name.replace(/^sol-cache-file-/, 'sol:'), content)
+      window.localStorage.setItem(name.replace(/^sol-cache-file-/, prefix), content)
```

Comment:
This is broken I think. `sol-cache-file-*` should only be renamed to `sol:`, it doesn't make sense under `sol-ui:` or any other prefix.